content/home/card_13/en-US.md

```diff
@@ -1,11 +1,11 @@
 ---
 # card 13
-title: "Mozilla Research Grant recipients announced"
-image: "grants.jpg"
-link_url: "https://blog.mozilla.org/blog/2018/11/28/mozilla-funds-research-grants-in-four-areas/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card"
-tag_label: "Grants"
+title: "Are you a developer?"
+image: "developer-edition.png"
+link_url: "https://www.mozilla.org/firefox/developer/"
```

Comment:
Note: I thought it made sense just to use a full prod URL for internal links like this? Once the editorial team use this admin, I think it will be pretty impossible for them to guess and internal url name like `firefox.developer.index`. Trying to do this would probably make it really easy to break the page on dev and have no idea why.

Reply:
It should also be able to just be "/firefox/developer/" so it will work to link to the page on whatever server it is. This may not work with our current implementation, but we could check for `url.startswith('/')` if that would be desirable.

Reply:
Are you suggesting I should change this to a relative URL here and now?